aat/src/resources/features/F-018/F-018.feature

```diff
@@ -0,0 +1,62 @@
+@F-018
+Feature: F-018: CCD Case Activity Api :: Add Activity
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+
+  @S-001
+  Scenario: should invoke activity service And return response on successful requests
```

Comment:
must successfully add an activity with a correct and complete invocation

Reply:
updated